lockbox-ios/Common/AppDelegate.swift

```diff
@@ -25,7 +25,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, willFinishLaunchingWithOptions
                      launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
         if isFirstRun {
-            KeychainWrapper.wipeKeychain()
+            KeychainWrapper.sharedAppContainerKeychain.clearAllValues(for: .account)
```

Comment:
IIUIC, we should be able to clear values for `.database` here as well.

Reply:
Agreed